src/pyOpenMS/pxds/DefaultParamHandler.pxd

```diff
@@ -12,8 +12,8 @@ cdef extern from "<OpenMS/DATASTRUCTURES/DefaultParamHandler.h>" namespace "Open
         # DefaultParamHandler(DefaultParamHandler & rhs) nogil except +
         # libcpp_vector[ String ] getSubsections() nogil except +
 
-        void setParameters(Param &param)  nogil except +
-        Param getParameters()  nogil except +
-        Param getDefaults()  nogil except +
-        String getName()  nogil except +
-        void setName(const String&)  nogil except +
+        void setParameters(Param &param)  nogil except + # wrap-doc:Sets the parameters
+        Param getParameters()  nogil except + # wrap-doc:Non-mutable access to the parameters
```

Comment:
@jpfeuffer: I am not sure about the non-mutable, mutable access - if that is the same in python as in C++ - do you know by chance?

Reply:
Nonmutable should be fine. The question if python developers understand that. Or if they think the object itself is nonmutable.

Reply:
I think we should remove "Non-mutable" from the doc as in python the return type will be immutable anyway, right?